x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_table.tsx

```diff
@@ -83,8 +84,7 @@ export const EnginesTable: React.FC<EnginesTableProps> = ({
       ),
       dataType: 'string',
       render: (dateString: string) => (
-        // e.g., Jan 1, 1970
-        <FormattedDate value={new Date(dateString)} year="numeric" month="short" day="numeric" />
+        <FormattedDateTime date={new Date(dateString)} hasTime={false} />
```

Comment:
I'm pretty 50/50 on this commit that adds a `hasTime` flag to the `<FormattedDateTime />` helper.

- Pros: It's useful for keeping the Engines table datestamp formatting DRY (if we ever change our date format, we can do it in 1 place and everything gets updated at once)
- Cons: It also adds a bit of cruft to the `<FormattedDateTime />` helper, maybe feels a little weird?

One alternative option might be to export the `year="numeric" month="short" day="numeric"` props as an obj const and import that here instead. I don't have a super strong preference either way so LMK what y'all think